File: src/cassandra.js
const {Client} = require("cassandra-driver");

async function run() {
    const client = new Client({
        cloud: {
            secureConnectBundle: "D:\\projects\\CS550\\PROJECT\\MongoDB\\secure-connect-cs550.zip",
        },
        credentials: {username: "burak", password: "burakkara"},
    });

    await client.connect();

    // Execute a query
    const rs = await client.execute("SELECT * FROM system.local");
    console.log(`Your cluster returned ${rs.rowLength} row(s)`);

    await client.shutdown();
}

// Run the async function
run();

File: src/cassandra.py
import cassandra as cc
#
# from cassandra.cluster import Cluster
# from cassandra.auth import PlainTextAuthProvider

cloud_config = {
    'secure_connect_bundle': 'D:\\projects\\CS550\PROJECT\\MongoDB\\secure-connect-cs550.zip'
}
auth_provider = cc.auth.PlainTextAuthProvider('burak', 'burakkara')
cluster = cc.cluster.Cluster(cloud=cloud_config, auth_provider=auth_provider)
session = cluster.connect()

row = session.execute("select release_version from system.local").one()
if row:
    print(row[0])
else:
    print("An error occurred.")
